### PR TITLE
Added Gradle config to README, updated dep version

### PR DESCRIPTION
**Pull request by BomBardyGamer**
Okay I believe I've got everything right this time, or as one would say, I nailed it. I've added a section to the README that shows an example of how to depend on the Paper API with Gradle, to help developers who choose Gradle over Maven. I've also updated the `<version>` tag in the Maven section and set the version in my new Gradle guide to `1.16.1-R0.1-SNAPSHOT` (as it should be), from `1.15.2-R0.1-SNAPSHOT`.

Again, please let me know if I screwed something up.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -42,4 +42,5 @@ rickyboy320 <rickw320@hotmail.com>
 DoNotSpamPls <7570108+DoNotSpamPls@users.noreply.github.com>
 JRoy <joshroy126@gmail.com>
 Robert Norman <spottedleaf@spottedleaf.dev>, <Spottedleaf@users.noreply.github.com>
+Callum Seabrook <callum.seabrook@prevarinite.com>
```
Comment (kennytv) [inline LICENSE.md]: Incredible.

File: README.md [hunk 2]
```diff
@@ -36,11 +36,29 @@ How To (Plugin Developers)
 <dependency>
     <groupId>com.destroystokyo.paper</groupId>
     <artifactId>paper-api</artifactId>
-    <version>1.15.2-R0.1-SNAPSHOT</version>
+    <version>1.16.1-R0.1-SNAPSHOT</version>
```
Comment (mdcfe) [inline README.md]: The master branch is not 1.16.1.

File: README.md [hunk 3]
```diff
@@ -36,11 +36,29 @@ How To (Plugin Developers)
...
+    maven {
+        url('https://papermc.io/repo/repository/maven-public/')
+    }
+}
+```
+
+ * Artifact:
+```groovy
+dependencies {
+    compileOnly('com.destroystokyo.paper:paper-api:1.16.1-R0.1-SNAPSHOT')
```
Comment (mdcfe) [inline README.md]: Ditto

**Review by mdcfe — CHANGES_REQUESTED** [review]

**Comment by BomBardyGamer** [comment]
Yeah you're right. Time for attempt 4?

File: README.md [hunk 4]
```diff
@@ -36,11 +36,29 @@ How To (Plugin Developers)
...
  </dependency>
  ```
 
+##### Or alternatively, with Gradle:
+
+ * Repository:
+```groovy
+repositories {
+    maven {
+        url('https://papermc.io/repo/repository/maven-public/')
```
Comment (JRoy) [inline README.md]:
```suggestion
        url 'https://papermc.io/repo/repository/maven-public/'
```

File: LICENSE.md (hunk 1 above)
Comment (BomBardyGamer) [reply LICENSE.md]: Well, thought I might as well as I'm more than happy to license this under MIT

**Comment by mdcfe** [comment]
You know you don't need to keep creating new PRs, right? You can push new commits to existing PR branches and it will update the PR, and the Paper team can squash these commits down to a single commit if they decide to merge your PR.

**Comment by wizjany** [comment]
please stop closing and re-opening new pull requests. it creates unnecessary spam. you can continue adding commits to this PR, and squash if needed.

edit: md ninja'd me

File: README.md (hunk 3 above)
Comment (JRoy) [inline README.md]:
```suggestion
    compileOnly 'com.destroystokyo.paper:paper-api:1.16.1-R0.1-SNAPSHOT'
```

File: LICENSE.md (hunk 1 above)
Comment (JRoy) [inline LICENSE.md]:
```suggestion
```

**Review by JRoy — CHANGES_REQUESTED** [review]
`master` isn't 1.16 yet, so targeting the master branch here is wrong. Also, you aren't providing patches so don't add your name to the `LICENSE.md` since the readme isn't code

File: LICENSE.md (hunk 1 above)
Comment (BomBardyGamer) [reply LICENSE.md]: What's the issue with that? I thought you just had to put your name in there to license your stuff under MIT... or am I missing something.

Comment (JRoy) [reply LICENSE.md]: YOU. CANNOT. LICENSE. YOUR. CHANGES. TO. THE. README. IT'S. ONLY. PATCHES.

**Comment by BomBardyGamer** [comment]
> please stop closing and re-opening new pull requests. it creates unnecessary spam. you can continue adding commits to this PR, and squash if needed.
> 
> edit: md ninja'd me

Didn't know that until I googled it a minute ago, thanks

File: README.md (hunk 3 above)
Comment (BomBardyGamer) [reply README.md]: That actually wasn't my idea, it was suggested by @Proximyst

File: README.md (hunk 4 above)
Comment (BomBardyGamer) [reply README.md]: Same as below

File: LICENSE.md (hunk 1 above)
Comment (kennytv) [reply LICENSE.md]: Yours truly licensing 10 lines of generic important statements

Comment (BomBardyGamer) [reply LICENSE.md]: Okay, okay, I'll change it hold on. Didn't know, my apologies.

File: README.md (hunk 2 above)
Comment (BomBardyGamer) [reply README.md]: Fixed, reverted back to 1.15.2-R0.1-SNAPSHOT, my bad.

File: README.md (hunk 3 above)
Comment (BomBardyGamer) [reply README.md]: Same as above.

**Comment by BomBardyGamer** [comment]
Should all be fixed now, let me know if you guys find anything else :slightly_smiling_face:

**Comment by JRoy** [comment]
You're still targeting master, needs to target `ver/1.16`

**Comment by BomBardyGamer** [comment]
My apologies, how do I change the branch on existing commits? or do I have to revert and re-commit

**Comment by DenWav** [comment]
You can't change the branch a PR is targeting. You have to open a new PR.

**Comment by DenWav** [comment]
Apparently I was wrong. That's neat.

**Comment by zachbr** [comment]
I just changed the base, but you'll have to fix some of the conflicts that snuck in somehow?
I think all you care about is the readme.

**Comment by BomBardyGamer** [comment]
Okay thanks, yeah the commits are all over the place since I had to fix a couple of things I screwed up in the initial commit. Any idea how to change those commits?

**Comment by kennytv** [comment]
Search for resetting, rebasing, and force-pushing

**Comment by clrxbl** [comment]
bruh moment

**Comment by BomBardyGamer** [comment]
Okay, so I've gotta reset the commits back to the latest commit from Aikar, then re-commit on the ver/1.16 branch. On it.

The only problem with that is if I reset softly it seems to give me many patch files that just apparently don't count as having been committed any more.

**Comment by kennytv** [comment]
Hard reset and do a new commit with your changes then, shouldn't be too hard to copy paste a bunch of text again

**Comment by BomBardyGamer** [comment]
Still seems to give me like 6 locally changed files if I hard reset to the latest commit before mine. :thinking:

**Comment by BomBardyGamer** [comment]
Just me being stupid and trying to reset back to the ver/1.16 branch

**Comment by BomBardyGamer** [comment]
Sorted. Am I gonna need a new PR to change the merge branch?

**Comment by zachbr** [comment]
No, just make sure that it looks how you want locally and force push over this branch.

**Comment by BomBardyGamer** [comment]
Can't seem to change the branch it's merging from, since my commit is now on ver/1.16 on my fork.

**Pull request closed without merging**